Require the needed files explicitly. Otherwise tests might break if this
test is found first.

--- test/testeventbuilder.rb.~1~	2006-12-24 20:29:01.000000000 +0100
+++ test/testeventbuilder.rb	2010-06-26 15:43:50.853708583 +0200
@@ -1,3 +1,6 @@
+require 'test/unit'
+require 'builder'
+
 class TestEvents < Test::Unit::TestCase
 
   class Target
